src/lib/stream-to-async-iterable.js

```diff
@@ -1,6 +1,19 @@
 'use strict'
 
-module.exports = function toIterable (body) {
+module.exports = function toAsyncIterable (res) {
+  const { body } = res
+
+  // An env where res.body getter for ReadableStream with getReader
+  // is not supported, for example in React Native
+  if (!body) {
+    if (res.arrayBuffer) {
+      return (async function * () {
+        const arrayBuffer = await res.arrayBuffer()
+        yield arrayBuffer
+      })()
+    }
```

Comment:
Can you please handle the case where `body` but there's no `res.arrayBuffer` - e.g. throw an error with an informative message, otherwise execution will continue and the code will try to access `body[Symbol.asyncIterator]` and fail.

Reply:
@achingbrain do you mean where body is undefined but there's no `res.arrayBuffer`? I'll assume that's what you meant.

Good call, we should handle that case. I'll add a test case for it too.

Reply:
@achingbrain do you mean where body is undefined but there's no `res.arrayBuffer`? I'll assume that's what you meant.

Good call, we should handle that case. I'll add a test case for it too.

Reply:
Addressed in this commit https://github.com/pcowgill/js-ipfs-http-client/commit/74c286ed08862fc7775a2f5ba51a44248faebcef